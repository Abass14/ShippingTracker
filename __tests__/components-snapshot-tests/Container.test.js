import ShallowRenderer from 'react-shallow-renderer';
import Container from '../../src/app/components/container';

it('should render AppContainer correctly', () => {
    const renderer = new ShallowRenderer()
    const tree = renderer.render(
            <Container />
    )
    expect(tree).toMatchSnapshot()
})