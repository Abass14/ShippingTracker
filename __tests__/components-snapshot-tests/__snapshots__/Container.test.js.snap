// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AppContainer correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#fff",
        "flex": 1,
        "gap": 20,
      },
      {
        "backgroundColor": "#fff",
      },
    ]
  }
>
  <StatusBar
    backgroundColor="#fff"
    barStyle="dark-content"
  />
  <RCTSafeAreaView
    style={
      {
        "backgroundColor": "#fff",
        "flex": 1,
        "gap": 20,
      }
    }
  >
    <KeyboardAvoidingView
      behavior="padding"
      style={
        [
          {
            "backgroundColor": "#fff",
            "flex": 1,
            "gap": 20,
          },
          {
            "paddingHorizontal": 20,
          },
        ]
      }
    />
  </RCTSafeAreaView>
</View>
`;
